depresolve/depresolve_test.go

```diff
@@ -0,0 +1,45 @@
+// +build nettest
+
+package depresolve
+
+import (
+	"reflect"
+	"testing"
+
+	"sourcegraph.com/sourcegraph/srclib/dep"
+)
+
+// TestResolveCustomImportPath tests the behavior of ResolveCustomImportPath
+// when called on some common Go package import paths.
+func TestResolveCustomImportPath(t *testing.T) {
+
+	testCases := []struct {
```

Comment:
Nitpick, remove the blank line above and call this `tests`. It's more consistent with how this is commonly written. E.g., see https://sourcegraph.com/sourcegraph/sourcegraph@master/-/def/GoPackage/sourcegraph.com/sourcegraph/sourcegraph/app/internal/canonicalurl/-/TestFromURL/tests.